libs/shared/otel/src/lib/node-tracing.ts

```diff
@@ -5,20 +5,30 @@
 import api from '@opentelemetry/api';
 import { suppressTracing } from '@opentelemetry/core';
 import { ILogger } from '@fxa/shared/log';
-import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
-import { registerInstrumentations } from '@opentelemetry/instrumentation';
-import { NodeTracerProvider } from '@opentelemetry/sdk-trace-node';
 import {
+  TracingOpts,
   checkSampleRate,
   checkServiceName,
-  TracingOpts,
   logType,
 } from './config';
-import { addConsoleExporter } from './exporters/fxa-console';
-import { addGcpTraceExporter } from './exporters/fxa-gcp';
-import { addOtlpTraceExporter } from './exporters/fxa-otlp';
+import {
+  NodeTracerProvider,
+  TraceIdRatioBasedSampler,
+  ParentBasedSampler,
+  SpanProcessor,
+  BatchSpanProcessor,
+  SimpleSpanProcessor,
+  SpanExporter,
+} from '@opentelemetry/sdk-trace-node';
+import { ATTR_SERVICE_NAME } from '@opentelemetry/semantic-conventions';
+import { resourceFromAttributes } from '@opentelemetry/resources';
+import { getConsoleTraceExporter } from './exporters/fxa-console';
+import { getGcpTraceExporter } from './exporters/fxa-gcp';
+import { getOtlpTraceExporter } from './exporters/fxa-otlp';
 import { createPiiFilter } from './pii-filters';
-import { createNodeProvider } from './providers/node-provider';
+import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+// import { NodeTracingInitializer } from './node-tracing-initializer';
 
 export const TRACER_NAME = 'fxa-tracer';
```

Comment:
Make this 'fxa' to be consistent with previous version.

Reply:
Yep

Reply:
But when I looked at the history, it looked it was originally 'fxa'... Maybe I missed something... main thing is we are consistent with the name.